glue_exp/tools/zoom_buttons/zoom_buttons.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+from glue.viewers.common.qt.mouse_mode import MouseMode
+
+try:
+    from glue.external.qt import QtGui
+except ImportError:
+    from qtpy import QtGui
+
+__all__ = ['ZoomButtonsTool']
+
+ROOT = os.path.dirname(__file__)
+
+
+class ZoomButtonsTool(object):
+
+    def __init__(self, widget=None):
+        self.widget = widget
+
+    def _get_modes(self, axes):
+        self._zoomin = ZoomInMode(axes)
+        self._zoomout = ZoomOutMode(axes)
+        return [self._zoomin, self._zoomout]
+
+    def _display_data_hook(self, data):
+        pass
+
+    def close(self):
+        pass
+
+
+class ZoomInMode(MouseMode):
+    """Buttom to zoom in."""
+
+    def __init__(self, *args, **kwargs):
+        super(ZoomInMode, self).__init__(*args, **kwargs)
+
+        self.icon = QtGui.QIcon(os.path.join(ROOT, 'glue_zoomin.png'))
+        self.mode_id = 'Zoom In'
+        self.action_text = 'Zoom In'
+        self.tool_tip = 'Zoom in'
+
+        # TODO: Pressing + should also zoom in.
+        self.shortcut = '+'
```

Comment:
Update: It only works *once*! I press "+", it zoomed in. Then the button is stuck in "activated" mode. I have to manually deactivate it by pressing the button, then it would work again. It wouldn't work if another button is activated (like the contour selection). So, this is not a pure key-binding as I have envisioned.